tools/internal_ci/macos/pull_request/grpc_distribtests_php.cfg

```diff
@@ -15,7 +15,7 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/macos/grpc_distribtests_php.sh"
+build_file: "grpc/tools/internal_ci/macos/grpc_distribtests_ruby.sh"
```

Comment:
s/ruby/php/?

Reply:
this is from the "DO NOT MERGE" commit (to kidnap the job and get and adhoc test result). I'll remove before merging.

Reply:
removed.